### PR TITLE
feat(relationships): add "without navigation property" section

### PR DESCRIPTION
**Pull request by Odonno**
related to #1209

### CODE REVIEW

File: samples/core/Modeling/FluentAPI/Samples/Relationships/NoNavigation.cs [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        public string Url { get; set; }
+    }
+
+    public class Post
+    {
+        public int PostId { get; set; }
+        public string Title { get; set; }
+        public string Content { get; set; }
+
+        public int PostForeignKey { get; set; }
```
Comment (divega) [inline samples/core/Modeling/FluentAPI/Samples/Relationships/NoNavigation.cs]: The name of the FK property is a bit strange. Can we changed to be just BlogId?

File: entity-framework/core/modeling/relationships.md [hunk 2]
```diff
@@ -135,6 +135,12 @@ You can use the string overload of `HasForeignKey(...)` to configure a shadow pr
 
 [!code-csharp[Main](../../../samples/core/Modeling/FluentAPI/Samples/Relationships/ShadowForeignKey.cs#Sample)]
 
+### Without Navigation Property
```
Comment (divega) [inline entity-framework/core/modeling/relationships.md]: @AndriySvyryd @ajcvickers can you think of any pitfalls worth mentioning here?

Comment (ajcvickers) [reply entity-framework/core/modeling/relationships.md]: Looks good. The only pitfall I can think of is the obvious one--there's no navigation properties to use!

File: samples/core/Modeling/FluentAPI/Samples/Relationships/NoNavigation.cs (hunk 1 above)
Comment (Odonno) [reply samples/core/Modeling/FluentAPI/Samples/Relationships/NoNavigation.cs]: Of course.

**Comment by divega** [comment]
Thanks a lot @Odonno!

**Pull request merged**